src/components/NewCommentForm.tsx

```diff
@@ -1,99 +1,195 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Comment } from '../types/Comment';
+import { addComment } from '../api/api';
+
+type Props = {
+  postId: number;
+  onAddNewComment: (newComment: Comment) => void;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  postId,
+  onAddNewComment,
+}) => {
+  const formValues = {
+    postId,
+    name: '',
+    email: '',
+    body: '',
+  };
+
```

Comment:
Move out of component
```suggestion

  const formValues = {
    postId,
    name: '',
    email: '',
    body: '',
  };

export const NewCommentForm: React.FC<Props> = ({
  postId,
  onAddNewComment,
}) => {

```